features/backoffice_deregister_exemption.feature

```diff
@@ -0,0 +1,21 @@
+@backoffice
+Feature: Back office user deregisters a waste exemption activity from a registration
+  As a back office user
+  I want to deregister a waste exemption
+  So that the exemption is deregistered for a valid registration with mutlipe exemptions
+
+  Background:
+    Given I am an internal user
+    And I have a valid username and password
+    And I complete a registration
+
+  @happy_path
+  Scenario: Deregistration of an exemption for an active registration by a NCCC user
```

Comment:
I think we can give more meaning to this feature if we refer specifically to ceasing and/or revoking an exemption. So the steps would read

```gherkin
When I [cease] an exemption
Then I can see the exemption status is [Ceased]
And when I [cease] the last active exemption
Then I can see the registration itself is [Ceased]
```

The things I've put in brackets could be passed as arguments to the steps, so they would still be reusable, but using the actual terms rather than the generic *deregistered/deregister* reads better and gives us more scope for testing different scenarios later.